app/lib/shared/urls.dart

```diff
@@ -154,32 +154,34 @@ String pkgDocUrl(
   String? version,
   bool includeHost = false,
   String? relativePath,
-  bool omitTrailingSlash = false,
   bool isLatest = false,
 }) {
+  final relativePathSegments = relativePath == null
+      ? const <String>[]
+      : Uri.parse(relativePath)
```

Comment:
Are we sure relativePath is always relative? Otherwise we might get some weird results here.

Reply:
`relativePath` comes from the "rest" part of the requested URL parsing after `/documentation/<package>/<version>/<relativePath>`.